www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx

```diff
@@ -15,7 +15,7 @@ const useStyles = makeStyles((theme) => ({
     marginLeft: theme.spacing(1),
   },
   command: {
-    fontWeight: theme.typography.fontWeightBold,
+    fontWeight: 'bold',
```

Comment:
```suggestion
    fontWeight: theme.typography.fontWeightBold,
```

Reply:
I get  typing issue if i set theme.typo.... He don't accept string | undefined

Reply:
As BDA said , there is a typing problem, in waiting to resolve issue , i fixe like that temporaly